libraries/Microsoft.Bot.StreamingExtensions/RequestEx.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+
+namespace Microsoft.Bot.StreamingExtensions
+{
+    /// <summary>
+    /// Implementation split between Request and RequestEx.
+    /// The basic request type sent over Bot Framework Protocol 3 with Streaming Extensions transports,
+    /// equivalent to HTTP request messages.
+    /// </summary>
+    public partial class Request
+    {
+        /// <summary>
+        /// Verb used by requests to get resources hosted on a remote server.
+        /// </summary>
+        public const string GET = "GET";
+
+        /// <summary>
+        /// Verb used by requests posting data to a remote server.
+        /// </summary>
+        public const string POST = "POST";
+
+        /// <summary>
+        /// Verb used by requests putting updated data on a remote server.
+        /// </summary>
+        public const string PUT = "PUT";
+
+        /// <summary>
+        /// Verb used by requests to delete data hosted on a remote server.
+        /// </summary>
+        public const string DELETE = "DELETE";
+
+        /// <summary>
+        /// Creates a <see cref="Request"/> to get resources hosted on a remote server.
+        /// </summary>
+        /// <param name="path">Optional path where the resource can be found on the remote server.</param>
+        /// <param name="body">Optional body to send to the remote server.</param>
+        /// <returns>A <see cref="Request"/> with appropriate status code and body.</returns>
+        public static Request CreateGet(string path = null, HttpContent body = null)
+        {
```

Comment:
Do we need to many null options here? This is effectivly a factory method, and should just be "Create"

Reply:
Yes, consumers use these.